source/includes/_platform-payments.md

```diff
@@ -158,198 +153,179 @@ Updates a specific set of stored account details. This will trigger an AML check
+| ----------------- | ------ | -------------------------------------------------- |
+| partyId           | string                                                      |
+| ukCompany.number  | string | The Company Number registered with Companies House.|
+| ukCompany.name    | string | Company Name                                       |
+
+
+## `GET /companies/{id} 🚧`
+```http
+GET /companies/COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502 HTTP/1.1
+Host: api-sandbox.goji.investments
+Content-Type: application/json
+Authorization: Basic ...
 ```
 
 ```http 
-HTTP/1.1 202 OK
+HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-  "id" : "3d9ca033-eb05-459f-9f70-1139d2e2b213"
+  "partyId": "COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502"
+  "ukCompany": {
+    "number": "00000000",
+    "name": "Fake Company LIMITED"
+  }
 }
 ```
-<aside class="notice">
-Prior to creating a wallet you need to have registered your company as the wallet holder using /wallet/company/registration
-</aside>
-
 ### Description
-Create a wallet
-### Request
-| Name       | Type   | Description     | Required |
-| ---------- | ------ | --------------- | -------- |
-| walletName | string | New wallet name | required |
+
+Retrieves details of a single company.
 
 ### Response
-| Name       | Type   | Description                                   |
-| ---------- | ------ | --------------------------------------------- |
-| id         | string | The unique identifier for the created wallet  |
+| Name              | Type   | Description                                        |
+| ----------------- | ------ | -------------------------------------------------- |
+| partyId           | string | Party ID for the company                           |
+| ukCompany.number  | string | The Company Number registered with Companies House.|
+| ukCompany.name    | string | Company Name                                       |
+
+## *Managing Wallets*
 
-## `GET /platformApi/wallet/{id}`
+
+## `POST /wallets 🚧`
 ```http
 
-GET /platformApi/wallet/{id} HTTP/1.1
+POST /wallets HTTP/1.1
 Host: api-sandbox.goji.investments
 Content-Type: application/json
 Authorization: Basic ...
 
+{
+  "ownerPartyId": "COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502",
+  "walletName" : "Housing Project 22"
+}
+
 ```
 
 ```http 
-HTTP/1.1 200 OK
+HTTP/1.1 202 Accepted
 Content-Type: application/json
 
 {
-  "id" : "walletId",
-  "bankAccountDetailsId" : "bankAccountDetailsId",
-  "bankDetails" : {
-    "accountName": "accountName",
-    "accountNumber": "accountNumber",
-    "sortCode": "sortCode"
-  },
-  "balance" : {
-    "amount" : 10.45,
-    "currency" : "GBP"
-  }
+  "id" : "3d9ca033-eb05-459f-9f70-1139d2e2b213",
+  "ownerPartyId": "COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502",
+  "walletName" : "Housing Project 22"
 }
 ```
+
+ℹ️ _Prior to creating a Wallet, the company who owns the money in the Wallet must be registered using [`POST /companies/{id}`](#payments-manager-post-companies)_
```

Comment:
You wouldn't specify with an ID - think that's the wrong displayed URL.

Reply:
Yes, typo, updating.